commands/permissions.go

```diff
@@ -132,3 +160,60 @@ Scheme Managed: no
 
 	return nil
 }
+
+func assignUsersCmdF(c client.Client, cmd *cobra.Command, args []string) error {
+	role, response := c.GetRoleByName(args[0])
+	if response.Error != nil {
+		return response.Error
+	}
+
+	for _, username := range args[1:] {
+		user, response := c.GetUserByUsername(username, "")
```

Comment:
here we can use the `getUsersFromUserArgs` to allow for more ways to reference users

Reply:
Switched to it. Thanks.